docker-compose.test.yml

```diff
@@ -2,6 +2,7 @@ version: '3.4'
 services:
   netbox:
     image: ${IMAGE-netboxcommunity/netbox:latest}
+    restart: unless-stopped
```

Comment:
I'm hesitant to add these to the testing compose file. The reason is that I believe that during testing it should be considered a bug if either service crashes. So I'm keen to learn your your thoughts for adding it to this file.

Reply:
The policy on the test can be left out :) My reasoning to add it was to keep it identical to the production version.

Reply:
I have removed the policy on the test compose file